src/types/Filter.ts

```diff
@@ -0,0 +1,3 @@
+export enum Filter {
+  All, Active, Completed,
+}
```

Comment:
this syntax looks better in my opinion
```suggestion
export enum Filter {
  All, 
  Active, 
  Completed,
}
```